examples/literate_markdown.jl

```diff
@@ -2,18 +2,7 @@
 
 # To create an example using CLIMA, please use [Literate.jl](https://github.com/fredrikekre/Literate.jl), and consult the [Literate documentation](https://fredrikekre.github.io/Literate.jl/stable/) for questions.
 #
-# For now, all literate examples are held in the `literate_examples` directory, and generated with the following code in `docs/make.jl`:
-
-using Literate
-const examples_directory = pwd() * "/literate_examples/"
-const output_directory = pwd() * "/generated/"
-
-examples = readdir(examples_directory)
-
-for example in examples
```

Comment:
@leios I don't know that we should expect users to add/modify these lines. I originally thought that contributors will simply add literate examples to the `CLIMA/examples` directory, and the docs will automatically pick them up. The only additional thing that's needed is for the .md files to be referenced, but this could be done automatically as well.

Reply:
You are right, they shouldn't change these lines. I just put them there to explain that everything was being picked up from a single directory. If I'm honest, this file was meant as a placeholder for adding things later.